holidays/countries/trinidad_and_tobago.py

```diff
@@ -0,0 +1,197 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from datetime import date
+from gettext import gettext as tr
+
+from holidays.calendars import _CustomHinduHolidays, _CustomIslamicHolidays
+from holidays.calendars.gregorian import MAR, APR, MAY, JUN, JUL, AUG, OCT, NOV
+from holidays.constants import OPTIONAL, PUBLIC
+from holidays.groups import (
+    ChristianHolidays,
+    HinduCalendarHolidays,
+    InternationalHolidays,
+    IslamicHolidays,
+)
+from holidays.observed_holiday_base import (
+    ObservedHolidayBase,
+    SUN_TO_NEXT_WORKDAY,
+    SAT_SUN_TO_NEXT_WORKDAY,
+    WORKDAY_TO_NEXT_WORKDAY,
+)
+
+
+class TrinidadAndTobago(
+    ObservedHolidayBase,
+    ChristianHolidays,
+    HinduCalendarHolidays,
+    InternationalHolidays,
+    IslamicHolidays,
+):
+    """Trinidad and Tobago holidays.
+
+    References:
+        * <https://web.archive.org/web/20250429121623/https://otp.tt/trinidad-and-tobago/national-holidays-and-awards/>
+        * <https://web.archive.org/web/20250429120744/https://laws.gov.tt/ttdll-web2/revision/list?offset=380&q=&currentid=322>
+        * <https://web.archive.org/web/20250429120707/https://www.guardian.co.tt/article-6.2.448640.3e9e366923>
+        * <https://web.archive.org/web/20250429120548/https://www.guardian.co.tt/article-6.2.410311.3fe66fb00f>
+        * <http://archive.today/2025.04.29-121801/https://www.nalis.gov.tt/resources/tt-content-guide/labour-day/>
+        * <https://web.archive.org/web/20250429120701/https://www.facebook.com/groups/191766699268/posts/10160832951274269/>
+        * <https://web.archive.org/save/https://www.facebook.com/100064996051675/posts/1101755938667598/>
```

Comment:
Sorry about the wrong link earlier 😔 
```suggestion
        * <https://web.archive.org/web/20250429142054/https://www.facebook.com/plugins/post.php?href=https%3A%2F%2Fwww.facebook.com%2F100064996051675%2Fposts%2F1101755938667598%2F>
```